docs/content/en/about/what-is-eosc.md

```diff
@@ -17,3 +17,9 @@ aliases: []
 toc: true
 ---
 
+eosc is a multi-platform command line development tool for securely storing your private keys and interacting with any EOSIO
```

Comment:
Do we want to put emphasis on eosc so that it is rendered as: `eosc` or *eosc* or **eosc**?